029-IoTEdge/README.md

```diff
@@ -0,0 +1,88 @@
+
+So how does this extend to the Azure IoT Edge space and how can this help?  IoT Edge offers capabilities to run custom 'code' or commercially supported [modules](https://docs.microsoft.com/en-us/azure/iot-edge/iot-edge-modules?view=iotedge-2018-06) on the compute appliance to perform tasks or to be triggered by an external event.  Microsoft makes available [Industrial IoT modules](https://azure.github.io/Industrial-IoT/) that can be deployed to IoT Edge devices deployed to plants that allow these devices to easily extract and interact or be triggered by data/state changes on PLCs using OPC!  This OpenHack will involve building a solution on how this can be leveraged to solve many common engineering scenarios in the industrial control space.
+
+## Background
+
+Contoso has a sophisticated manufacturing environment distributed across many plants in many countries.  They use a number of PLCs in these plants but have a common vendor they use for an OPC/UA server to capture data for their plant historian.  They would like to take advantage of real-time analytics capabilities of the cloud to stream their production process control data generated by sensors and instrumentation captured by their PLCs in the cloud.  Their security team is concerned about opening their OPC server up to public internet connectivity but their quality and maintenance teams would both like to use cloud ML and time-series reporting capabilities to analyze data emitted from PLCs.  Your goal as an architect is to help them plan a solution on how to safely extract this data using existing compute resources while addressing their concerns for security.   
+
+This hack includes presentations that feature lectures introducing key topics associated with each challenge. It is recommended that the host present each lecture before attendees kick off that challenge.
+
+## Learning Objectives
+
+
+In this hack you will solve common challenges for companies planning to use Azure IoT in Industrial IoT scenarios. This includes:
+
+1. Deploying and configuring the IoT Edge runtime
+1. Deploying modules to the running IoT Edge device
+1. Managing/monitoring/configuring IoT Edge devices at scale from IoT Hub
+1. Configuring an OPC (simulator) to generate factory data
+1. Interfacing IoT Edge to the factory simulator to capture data and push to the cloud
+1. Consuming data published to the cloud for analysis and reporting
+
+## Challenges
+
+- Challenge 1: **[Deploy IoTHub/Edge](Student/Challenge-01.md)**
+  - Get familiar with the basic concepts of Azure IoT Hub and Azure IoT Edge.
+  - Iot Hub Creation
+  - Edge Device creation
+
+- Challenge 2: **[Deploy OPC Simulator](Student/Challenge-02.md)** 
+  - OPC Simulator to serve as our virtual 'factory'
+
+- Challenge 3: **[Deploy Industrial IoT solution](Student/Challenge-03.md)**
+  - Deploy Industrial IoT platform to the IoT Edge
+
+- Challenge 4: **[Route messages and do time-series analysis](Student/Challenge-04.md)** **RYAN/AMIT**
```

Comment:
You  need to remove everyone's name assignments on the challenges! eg: this line has **RYAN/AMIT**